.cirrus.yml

```diff
@@ -1,25 +1,37 @@
-root_task_template: &ROOT_TASK_TEMPLATE
-  # Don't run on release tags since it creates O(n^2) tasks where n is the
-  # number of plugins
-  only_if: $CIRRUS_TAG == ''
-  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR == ''
-  env:
-    INTEGRATION_TEST_PATH: "./packages/integration_test"
-    CHANNEL: "master" # Default to master when not explicitly set by a task.
-    PLUGIN_TOOLS: "dart run ./script/tool/lib/src/main.dart"
-  setup_script:
-    - flutter channel $CHANNEL
-    - flutter upgrade
+setup_tool_template: &SETUP_TOOL_TEMPLATE
+  setup_tool_script:
```

Comment:
Nit: how about tool_setup_script. Since setup is a noun, this sounds like it's about a setup tool to me with the current name.